dist/example_config.yaml

```diff
@@ -68,6 +68,13 @@ chihaya:
     # The maximum number of infohashes that can be scraped in one request.
     max_scrape_infohashes: 50
 
+    # The IP addresses from which a fullscrape will be allowed.
+    # Note: Fullscrapes are potentially expensive and may slow the tracker down.
+    # It is recommended to only allow this from trusted IPs.
+    # Whether fullscrapes are supported depends on the storage implementation.
+    fullscrape_whitelisted_ips:
+      - "127.0.0.1"
```

Comment:
Can we make these CIDRs instead of IPs? That'd make this significantly more configurable.

Reply:
Done